shared/util/forward-logs.native.js

```diff
@@ -18,19 +18,19 @@ function setupSource() {
   }
   forwarded = true
 
-  window.console.log = (...args) => {
-    localLog(...args)
-    logger.info(...args)
+  window.console.log = (a1, a2, a3, a4, a5, a6, a7, a8, a9, a10) => {
+    localLog(a1, a2, a3, a4, a5, a6, a7, a8, a9, a10)
+    logger.info(a1, a2, a3, a4, a5, a6, a7, a8, a9, a10)
   }
 
-  window.console.warn = (...args) => {
-    localWarn(...args)
-    logger.warn(...args)
+  window.console.warn = (a1, a2, a3, a4, a5, a6, a7, a8, a9, a10) => {
```

Comment:
this code in general is a hack (overriding console.log) and i think is something we want to move away from and more towards a real logger. but short term this fixes a real perf problem and is totally isolated to the implementation of this passthrough. nothing needs to know about this and it affects nothing else